cc/main.cc

```diff
@@ -204,6 +219,52 @@ void SelfPlay() {
   }
 }
 
+void SetMctsPlayerOptionsForEval(MctsPlayer::Options* options) {
+  options->inject_noise = false;
+  options->soft_pick = false;
+  options->random_symmetry = true;
+  options->resign_threshold = FLAGS_resign_threshold;
+  options->batch_size = FLAGS_batch_size;
+  options->komi = FLAGS_komi;
+  options->random_seed = FLAGS_seed;
+}
+
+void Eval() {
+  MctsPlayer::Options options;
+  SetMctsPlayerOptionsForEval(&options);
```

Comment:
I'd be tempted to just call ParseMctsPlayerOptionsFromFlags here, then override inject_noise, soft_pick and random_symmetry explicitly.

Reply:
Done.